cloudformation/cf10aws.yml

```
@@ -0,0 +1,365 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters: 
+
+# MyIP,create
+  MyIP:
+    Description: IP address allowed to access EC2
+    Type: String
+
+# KeyName,create
+  KeyName:
+    Description: The EC2 Key Pair to allow SSH access to the instance
+    Type: "AWS::EC2::KeyPair::KeyName"
+
+Resources: 
+
+# Vpc,create
+  Vpc10:
+    Type: AWS::EC2::VPC
+    Properties:
+      CidrBlock: 10.10.0.0/20
+      EnableDnsSupport: true
+      Tags:
+        - Key: Name
+          Value: Vpc10Cf
+
+# InternetGateWay,create
+  InternetGateWay10:
+    Type: AWS::EC2::InternetGateway
+    Properties:
+      Tags:
+        - Key: Name
+          Value: InternetGateWay10Cf
+
+  AttachGateway:
+    Type: AWS::EC2::VPCGatewayAttachment
+    Properties:
+      VpcId: !Ref Vpc10
+      InternetGatewayId: !Ref InternetGateWay10
+
+# Subnet,create
+  PublicSubnet11:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1a"
+      VpcId: !Ref Vpc10
+      CidrBlock: 10.10.0.0/23
+      Tags:
+        - Key: Name
+          Value: PublicSubnet11Cf
+
+  PrivateSubnet11:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1a"
+      VpcId: !Ref Vpc10
+      CidrBlock: 10.10.2.0/23
+      Tags:
+        - Key: Name
+          Value: PrivateSubnet11Cf
+
+  PublicSubnet12:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1c"
+      VpcId: !Ref Vpc10
+      CidrBlock: 10.10.4.0/23
+      Tags:
+        - Key: Name
+          Value: PublicSubnet12Cf
+
+  PrivateSubnet12:
+    Type: AWS::EC2::Subnet
+    Properties:
+      AvailabilityZone: "ap-northeast-1c"
+      VpcId: !Ref Vpc10
+      CidrBlock: 10.10.6.0/23
+      Tags:
+        - Key: Name
+          Value: PrivateSubnet12Cf
+
+# RouteTable,RouteTableAssociation,Route,create
+  RouteTable10Pus11:
+    Type: AWS::EC2::RouteTable
+    Properties:
+      VpcId: !Ref Vpc10
+      Tags:
+        - Key: Name
+          Value: RouteTable10Pus11Cf
+  
+  RouteTableAssociation10Pus11:
+    Type: AWS::EC2::SubnetRouteTableAssociation
+    Properties:
+      SubnetId: !Ref PublicSubnet11
+      RouteTableId: !Ref RouteTable10Pus11
+
+  Route10Pus11:
+    Type: AWS::EC2::Route
+    Properties:
+      RouteTableId: !Ref RouteTable10Pus11
+      DestinationCidrBlock: 0.0.0.0/0
+      GatewayId: !Ref InternetGateWay10
+
+  RouteTable10Prs11:
```

こちらも`Prs`がPrivateSubnetの略なら下記になるかと！
```suggestion
  RouteTable10PrS11:
```

---

ありがとうございます！
修正します！
ちなみに以前言われていたSSH接続のCidrIPを、Parameter化してセキュリティ対策を行ってみたんですが、言われていた内容と間違いはないですかね？

---

> ちなみに以前言われていたSSH接続のCidrIPを、Parameter化してセキュリティ対策を行ってみたんですが、言われていた内容と間違いはないですかね？

あってますよー！修正ナイスです！
ちなみに`MasterUserPassword` も平文で載せるのはNGなのでこちらもParameter化しちゃいましょう！

---

迷っていたところでした！
ありがとうございます！修正します！